clipboard/ios/Plugin/Clipboard.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import Capacitor
+
+@objc public class Clipboard: NSObject {
+    
+    @objc enum ContentType: Int {
+        case string
+        case url
+        case image
+    }
+    
+    @objc func write(_ label: ContentType, _ content: String) {
```

Comment:
Omitting parameter names is idiomatic for JavaScript and makes sense for the methods exposed on the Plugin wrapper. However, internal implementations should better conform to platform conventions. So, for example, a more idiomatic way to declare this method would be:
```suggestion
    @objc func write(content: String, ofType type: ContentType) {
```